@navikt/core/react/src/form/combobox/Input/Input.context.tsx

```diff
@@ -26,9 +25,13 @@ interface InputContextType extends FormFieldType {
   toggleOpenButtonRef: React.RefObject<HTMLButtonElement>;
 }
 
-const InputContext = createContext<InputContextType>({} as InputContextType);
+const [InputContextImplProvider, useInputContext] =
```

Comment:
Vet ikke helt om jeg forstår hva denne gjør 🤔 Hva står "Impl" for i `InputContextImplProvider`? Implementation? Kanskje vi kan vi et tydeligere navn.

Reply:
👍 Ja var bare et litt tilfeldig navn siden `InputContextProvider` allerede er definert rett under. Tar gjerne imot forslag!

Reply:
Fikset med en enkel `InputProvider as InputContextProvider` i export 🚀